dev-docs/workflows/marketplace-publishing.md

```diff
@@ -0,0 +1,33 @@
+# Publishing Marketplace Images
+
+Constellation release images need to be manually published to AWS and Azure marketplaces due to the lack of automation features.
+On GCP, marketplace image publishing is automated and takes place on release.
+
+This document explains how to perform the uploading on AWS and Azure.
+
+## AWS
+
+1. Log in to the [AWS marketplace management portal](https://aws.amazon.com/marketplace/management/) with your regular developer IAM account.
+2. Select "Products -> Server -> Constellation" in the top menu.
+3. Select "Versions" in the main menu and press "Add version".
+4. Fill in the form.
+    1. Enter the semantic version of the release (i.e. `vX.Y.Z`) as "Version title".
+    2. Set the version tag in "Release notes" to the same version.
+    3. For the "Amazon Machine Image (AMI) ID", enter the AMI ID of the release (SEV-SNP) image. This can be found in the regular
+       [AWS console](https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#Images:visibility=owned-by-me;search=:constellation-v2.2,:constellation-v;v=3;$case=tags:false%5C,client:false;$regex=tags:false%5C,client:false;sort=desc:creationDate).
+    4. For "IAM access role ARN", enter `arn:aws:iam::795746500882:role/constellation-marketplace-ingest`.
+5. Leave the other fields as they are and press "Add version".
+6. Wait for the [request](https://aws.amazon.com/marketplace/management/requests) to be processed and available before publishing the release.
+
+## Azure
+
+1. Log in to the [Microsoft partner center](https://partner.microsoft.com/en-us/dashboard/home) with your regular developer IAM account.
+2. Select "Marketplace offers -> Constellation -> Constellation Node" in the main menu.
```

Comment:
I don't have access to `Marketplace offers` - do we need to explain this here? Should every engineer have access?

Reply:
Not sure about this. I thought everyone would have access since I had. @m1ghtym0 do you know more?

Reply:
Yes, we probably only gave explicit access. Will check if we can give it to the devs group.